### PR TITLE
Callendar section

### PR DESCRIPTION
**Pull request by Romano78**
Gatsby Build successful.
Works on all browsers. 
Not sure if the animation for the Menu links is what you wanted but that's an easy fix if we need to change it.

### CODE REVIEW

**Comment by Romano78** [comment]
I think the checks have failed because we need to add the google Api key.

**Review by jeanpco — CHANGES_REQUESTED** [review]
1. Mobile Event layout / fonts / spacing / line divider all differs from the figma mockup. Please compare closely designs with prod site and notice differences.

Screen Shot 2020-10-04 at 12 05 53 PM

2. Too much spacing between calandar and first event.
Screen Shot 2020-10-04 at 12 05 45 PM

3. Desktop line divider and spacing is off.

Screen Shot 2020-10-04 at 12 03 44 PM

4. Desktop spacing of content is off + validate fonts with Figma, e.g: body font is wrong.
Screen Shot 2020-10-04 at 12 03 16 PM

**Review by jeanpco — CHANGES_REQUESTED** [review]
1. Try toggling between a date without event and with events and notice how the calendar itself changes size which creates some sort of jump. We can't have that.

Screen Shot 2020-10-04 at 11 57 42 AM

File: src/components/EventsCalendar/NavbarElement/styles.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+    @media (min-width: ${(props) => props.theme.breakpoints.l}) {
+      top: 56px;
+    }
+  }
+
+  button {
+    @media (min-width: ${(props) => props.theme.breakpoints.md}) {
+    }
+  }
```
Comment (jeanpco) [inline src/components/EventsCalendar/NavbarElement/styles.js:22]: doesnt look like we need this

File: src/components/Utilities/Title/styles.js [hunk 2]
```diff
@@ -88,6 +88,17 @@ export const TitleStyled = styled.p`
       font-size: ${theme.fonts.heading6};
     }
   }
+
+  &.calendarTitle {
```
Comment (jeanpco) [inline src/components/Utilities/Title/styles.js:105]: Try changing the name to something that could be used outside of the calendar

**Review by jeanpco — CHANGES_REQUESTED** [review]

**Pull request merged**